SpatialGDK/Documentation/content/porting-unreal-project-to-gdk.md

```diff
@@ -113,63 +128,68 @@ etc...
 
     > These lines ensure that the SpatialOS Unreal GDK can override Unreal's network replication.
     >
-    > If you ever need to run with Unreal networking for workflow or validation reasons, untick `Spatial Networking` in the `Play` drop down menu in the Unreal editor. This flag is maintained in your Unreal config files under `/Script/EngineSettings.GeneralProjectSettings` and is also valid for non-editor builds.
-    >
-    > You can easily switch between networking modes using this checkbox. As the SpatialOS Unreal GDK is in pre-alpha, switching back to Unreal's networking may be a viable way to work around some issues you may encounter, and may speed up your iteration. However, in Unreal networking mode you lose access to key multi-server features of the GDK, which may lead to erratic behavior.
+    > If you ever need to run with default Unreal networking for workflow or validation reasons, un-tick `Spatial Networking` in the `Play` drop down menu in the Unreal editor. This flag is maintained in your Unreal config file `<GameRoot>/Config/DefaultGame.ini` under `/Script/EngineSettings.GeneralProjectSettings` and is also valid for non-editor builds.
```

Comment:
Two edits here:

_____

“..or validation reasons, un-tick `Spatial Networking` in the `Play` drop down menu in the Unreal editor."
(this is written backwards to how you instruct someone, giving the specifics first. It also  uses the filename/path formatting rather than the UI formatting and uses "tick" which is British English (sorry, Josh, has to go :-( !!). )
-> 
..or validation reasons, you can switch networking in the Unreal Editor: from the Editor menu, select *Play* and from the drop-down menu, un-check `Spatial Networking` .

----
"This flag is maintained in your Unreal config file `<GameRoot>/Config/DefaultGame.ini` under `/Script/EngineSettings.GeneralProjectSettings` and is also valid for non-editor builds."
-  Slipped into passive voice (who or what maintains it? Do I need to do something?)
 - "Non-Editor" builds are command-line builds, I think?, so let's say what they are as it's helpful to be specific about the use (unless there's loads of other ways to build and they are too numerous to specify).
- Let's standardize the slashes.
- It's a setting in the Editor, I think, so we should call it that.

->
"This setting is valid for Editor and command-line builds. It is stored in your game project's Unreal config file; `<GameRoot>\Config\DefaultGame.ini` under `\Script\EngineSettings.GeneralProjectSettings`."

Reply:
That looks pretty good. Making the change.

Reply:
Note the first sentence is also back to front and not  so clear. I have suggested:
"..or validation reasons, you can switch networking in the Unreal Editor: from the Editor menu, select Play and from the drop-down menu un-check Spatial Networking ."

Reply:
Done so.